CONTRIBUTING.md

```diff
@@ -1,6 +1,34 @@
 # Contributing
 
-We love pull requests. If you have something you want to add or remove, please open a new pull request.
+If you have something you want to add or remove, please use the following agreed upon process.
+
+### Initial Proposal Creation
+ - Proposals are made by creating a GitHub [issue](https://help.github.com/articles/creating-an-issue/) or [pull request](https://help.github.com/articles/creating-a-pull-request/) (PR).
+ - Issues containing the proposal are tagged with the area(s) under which they fall using [GitHub labels](https://github.com/sul-dlss/DeveloperPlaybook/labels).
+ - The proposal is an appropriate granularity requirement to capture existing consensus.
+ - The proposal is of the form When <doing activity>, we [MUST|SHOULD] <use described approach>.
+ - Proposals are to be scoped to whether it applies universally, or can be decided on by individual project teams.
+ - The body of the issue must describe the rationale for why the entry is needed
+   - What is the danger if there isn't a best practice?
+   - How do people most effectively follow it?
+  
+### Gaining Consensus
```

Comment:
That's fine. Let's work on that pull request. I'm happy to help if you could suggest a broad outline or even specific elements that should be incorporated. As is, I feel like I'm in the dark and find it difficult to contribute towards something we can agree on.